internal/cmd/run/cypress.go

```diff
@@ -149,10 +151,15 @@ func runCypressInSauce(p cypress.Project, regio region.Region, tc testcomposer.C
 			ShowConsoleLog:     p.ShowConsoleLog,
 			ArtifactDownloader: &rs,
 			Reporters:          createReporters(p.Reporters),
+			SlackService:       &tc,
+			Notifier: &slack.Notifier{
+				Channels:  p.Notifications.Slack.Channels,
+				Framework: "cypress",
+				Metadata:  p.Sauce.Metadata,
+				TestEnv:   "sauce",
+			},
 		},
 	}
-
-	cleanCypressPackages(&p)
```

Comment:
Why removed? 🤔

Reply:
Thanks! It was removed unintentionally during rebase with the main branch. I returned it back.